frontend/src/components/TaskItem/index.tsx

```diff
@@ -53,7 +53,10 @@ export default function TaskListItem(props: GetTaskFormProps) {
                 onClick={(e) => {
                   e.stopPropagation();
                   e.preventDefault();
-                  // do other stuff here
+                  if (confirm(`Mark task "${props.task.name}" and all of its sub-tasks as complete?`)) {
```

Comment:
i can see this confirm dialog every time you check off a task being annoying, i see why its useful for when there are subtasks tho. maybe only show it if there are subtasks for the task being completed?

Reply:
I explored doing that, but I think that'll have to be something that's done after #74, since the GET Task endpoint does not (and in my opinion, should not) return its children. In the actual list, the GET Tasks endpoint will be used, which will actually have the hierarchy

Reply:
this is a good point, sounds good